### ISSUE TITLE
PTN000256565

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2063]
for pombe ned1

GO:1903508 | positive regulation of nucleic acid-templated transcription

from
http://www.informatics.jax.org/marker/MGI:1891340
http://www.informatics.jax.org/marker/MGI:1891341

ned1  (lipin)
a phosphatidate lipid phosphatase catalyzing the hydrolytic dephosphorylation of a phos- phatidate to a diacylglycerol. It controls cellular lipid homeostasis

nothing really to do with transcription, although the nuclear envelope deformations caused in mutants will certainly affect transcription, it isn't regulating transcription in  a WT cell.

**Comment by pgaudet** [comment]
The yeast gene is also some sort of tx coregulator: https://www.ncbi.nlm.nih.gov/pubmed/15889145

Had you also looked at that reference ?

**Comment by ValWood** [comment]
I don't think its really a txn regulator.

This  mutant is disturbing the nuclear membrane. During membrane stresses changes in inositol levels are detected, and this activates transcription of lipid biosynthesis genes. In a normal cell I don't think lipin is a transcriptional regulator. Thsi mutant  is causing a problem that is sensed  by some pathway and  lipid bio genes are turned on.
Certainly not enough info for GO for us.....because it  is most likely  just a phenotype.

(SGD did not make this annotation either)

**Comment by pgaudet** [comment]
They have 'transcription regulatory region DNA binding (IDA) ' 
https://www.yeastgenome.org/locus/S000004775

and looking at the paper cited (PMID:15889145) , including this part of the abstract 
>We show that Smp2 associates with the promoters of phospholipid biosynthetic enzymes in a Nem1-Spo7-dependent manner. Our data suggest that Smp2 is a critical factor in coordinating phospholipid biosynthesis at the nuclear/ER membrane with nuclear growth during the cell cycle.

I thought perhaps there was enough evidence from varied sources to make the annotations. 


Pascale

**Comment by ValWood** [comment]
I asked a couple of people just to be sure....

**Comment by pgaudet** [comment]
The experts contacted by @ValWood write 
> The budding yeast transcription circuit is not conserved outside budding yeasts (actually, a big chunk of inositol metabolism is different between the two clades). The mammalian data suggest a different mechanism and are not particularly direct either.

> Our own (unpublished) data indicate that all transcriptional changes in pombe lipin deletion are due to the loss of its PA->DAG catalytic activity (the situation seems different in japonicus). 

> So what am trying to say, it is probably OK to say that such regulation has been picked up in other species but the extent to which it can be generalized is not clear. As a disclaimer, I might have missed something!

So I'll remove the propagation from the yeast clade. 

Thanks, Pascale
